scripts/tests/relayed-curl-relay-restart.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+source "./scripts/tests/lib.sh"
+
+install_iptables_drop_rules
+trap remove_iptables_drop_rules EXIT # Cleanup after us
+
```

Comment:
Same thing here re: restart order.

This tests that the establishment of new data channels for a client that is already connected to the portal still works after a relay restart.

We could (should) certainly test that already-established channels function properly too.

Reply:
Updated to test both scenarios by restarting both with and without a binding

Reply:
> This tests that the establishment of new data channels for a client that is already connected to the portal still works after a relay restart.

Prior to a connection, there is no state on the client for a particular relay. We don't even know its IP. Thus I think the first relay restart doesn't do anything in regards to the client's state.

In other words, for the client, it makes no difference how many times you restart a relay before you first learn about it. It is the same state.

Restarting a relay _after_ is a good test case.

I think we should remove the first restart because it is confusing as to why it would matter.